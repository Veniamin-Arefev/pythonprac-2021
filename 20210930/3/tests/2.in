99, 62, -90, -47, -22, 99, 6, 12, 27, 14
72, -86, 90, 35, -7, -30, -20, -40, -66, 42
93, 44, -22, 3, -17, -11, 54, 59, 68, -13
-86, 76, -60, -54, 95, -27, 69, -47, 98, 43
-94, -72, -11, -65, 99, -39, 41, 2, 23, -40
26, 36, 84, 20, 81, -35, 10, -42, 75, 35
-35, 33, 97, 79, 43, -67, 29, 56, -68, -35
-72, -57, 67, -54, 15, -37, 50, -7, 47, -75
-6, -61, -58, -58, -99, -53, -23, 25, -13, 30
79, -24, 67, 80, 62, 61, 17, -76, -71, -19
49, 65, -38, 3, -32, 44, 72, 6, 65, -16
-16, 71, 52, 63, 98, -60, -18, 100, -92, 53
-40, -84, 0, -37, 34, 65, -9, -16, 70, 7
17, -3, 15, -21, -67, 79, -36, -89, 11, -88
87, 97, 6, -99, 69, -43, -51, -63, -39, 16
-41, 93, -67, -17, -38, 84, -38, -75, -18, 49
100, -12, -41, -4, -62, -4, 47, -97, 36, 99
-24, -12, -14, 77, -36, 68, 78, 8, -28, -11
1, 15, 36, 71, 70, -100, -14, 96, 63, 34
-35, -91, -60, -83, 35, 55, -23, 17, -57, -100