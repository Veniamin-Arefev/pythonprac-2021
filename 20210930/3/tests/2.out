[536, 24526, -8290, 10670, -707, -803, 6392, 8722, -6227, 9687]
[-56, -16652, -8231, -19236, -9830, 20994, 2057, -14034, 13938, -17726]
[8263, 9183, -378, 15906, -1535, -3653, 13766, 8480, 6668, 7924]
[12045, 9005, 7566, -1568, 25138, -33770, -11679, 11960, -13072, 16313]
[11568, -658, 3579, -7241, 5842, -17727, -3919, -7548, 1019, 11823]
[7018, 56, 4793, -9460, 18498, -8870, -7714, 4643, 6215, 403]
[4421, -9232, 6657, -4119, -1927, 10116, 1010, -12665, -936, -3116]
[4448, -9165, 5771, 2815, 2872, -12913, 695, -2237, 13973, 15904]
[-8387, -17156, -2532, 8809, -8023, -409, 10700, 11859, 1282, -8614]
[9946, 10622, -7814, -21993, -9955, 18868, -6541, -27995, 8747, -3121]
